libs/pavex/src/connection.rs

```diff
@@ -0,0 +1,17 @@
+//! Extract data concerning the HTTP connection.
+use std::net::SocketAddr;
+
+/// Information relating to the current undelying HTTP connection.
+///
+/// It includes the [peer address](SocketAddr).
+#[derive(Clone, Debug)]
+pub struct ConnectionInfo {
+    pub(crate) peer_addr: SocketAddr,
+}
+
+impl ConnectionInfo {
+    /// Returns the peer address.
```

Comment:
Let's add a little example here.

Reply:
A handler, accepting `ConnectionInfo`, and then printing the peer address? Or rejecting on all but a specific peer IP?

Reply:
Let's do the first, yup.

Reply:
https://github.com/LukeMathWalker/pavex/pull/169/commits/819557f5ff01a9c22338a94be3790ce0a8cd1de6